sdk/src/main/java/io/opentelemetry/sdk/metrics/export/OpenTelemetryMetricProducer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics.export;
+
+import io.opentelemetry.sdk.metrics.MeterSdkProvider;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import java.util.Collection;
+
+/** A {@link MetricProducer} that can read metrics from an instance of {@link MeterSdkProvider}. */
+public final class OpenTelemetryMetricProducer implements MetricProducer {
```

Comment:
this name feels weird to me in the context of the rest of the naming in the SDK. How about `SdkMetricProducer` or `MetricProducerSdk` to match the naming patterns everywhere else?

Reply:
Tried a different design, what do you think?

Reply:
I like the one-stop-shop approach. Let's go with the new design and see how it feels in the "real world" over the next week or so.  👍